src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -60,6 +57,7 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
         public event EventHandler PreSolutionBindingUpdated;
         public event EventHandler SolutionBindingUpdated;
 
+        public string CurrentSolutionName { get; private set; }
```

Comment:
This is only getting `set` and never `get`. It can be removed, the source of truth for the CurrentSolutionName should be the `ActiveSolutionTracker`